content/ko/docs/concepts/architecture/cri.md

```diff
@@ -0,0 +1,42 @@
+---
+title: 컨테이너 런타임 인터페이스(CRI)
+content_type: concept
+weight: 50
+---
+
+<!-- overview -->
+
+컨테이너 런타임 인터페이스(CRI)는 클러스터 컴포넌트를 다시 컴파일하지 않아도 Kubelet이 다양한 컨테이너 런타임을 사용할 수 있도록 하는 플러그인 인터페이스다.
+
+클러스터의 모든 노드에 동작 중인
+{{<glossary_tooltip text="컨테이너 런타임" term_id="container-runtime">}}이 존재해야, {{< glossary_tooltip text="kubelet" term_id="kubelet" >}}이
+{{< glossary_tooltip text="Pods" term_id="pod" >}} 과 컨테이너들을 구동할 수 있다.
+
+Kubelet과 컨테이너 런타임 간의 통신을 위한 중심 프로토콜이다.
+
+쿠버네티스 컨테이너 런타임은 클러스터 컴포넌트, Kubelet 과 컨테이너 런타임 간의 통신을 위한 중심 gRPC 프로토콜을 정의한다.
```

Comment:
This was originally written with the shortcode in English version.

```
{{< glossary_definition term_id="container-runtime-interface" length="all" >}}
```

But, I couldn't figure out how to translate the original parts in natuaral way. So I just translated in raw. If there is another way, please let me know :)

Reply:
@anencore94 Thank you for the contribution ! :) 

`{{< glossary_definition term_id="container-runtime-interface" length="all" >}}` can be translated by translating `content/en/docs/reference/glossary/container-runtime-interface.md` (adding a new Korean doc `content/ko/docs/reference/glossary/container-runtime-interface.md`)

Translating it in raw is not a conventional way ;) So, please revert this section to remain it as a code. Even though `{{< glossary_definition term_id="container-runtime-interface" length="all" >}}` shows English.

We can add content/**ko**/docs/reference/glossary/container-runtime-interface.md in a different PR. 
We can also include tranlsated `container-runtime-interface.md` in this PR as well. This will be a better option. but, I think it depends on contributor's decision. (cc: @jmyung)

In addition, you can use Korean comments :0
We need to use Korean for review anyway.

Reply:
| We can also include tranlsated container-runtime-interface.md in this PR as well. 

이 방식으로 이번 PR에서 함께 추가하도록 하겠습니다. 친절한 리뷰 감사드립니다! @jmyung @seokho-son

Reply:
코멘트 주신대로 수정 반영하였습니다. re-review 해주시면 감사하겠습니다!

Reply:
> 참고로 main 브랜치에서 내용이 변경되었습니다. 이번 PR은 그대로 진행해주시면 될 것 같아요.

넵 그럼 해당 부분은 다음 PR 에서 반영하겠습니다